src/app/edit-pet/edit-pet.component.html

```diff
@@ -1,10 +1,9 @@
 <div>
   <h1>Edit Pet</h1>
   <hr>
-  <div class="spinner" *ngIf="!this.loaded">
-    <mat-spinner></mat-spinner>
+  <div *ngIf="this.pet$">
```

Comment:
The observable will always be truthy

Reply:
I just removed the div entirely since it isn't doing anything.